quarkchain/core.py

```diff
@@ -1102,7 +1102,7 @@ def __init__(
         self.is_from_root_chain = is_from_root_chain
 
 
-class CrossShardTransactionDeposit(Serializable):
+class CrossShardTransactionDepositV0(Serializable):
```

Comment:
For versioning, I would suggest
- CrossShardTransactionDepositDeprecated as V0
- CrossShardTransactionDeposit as V1
- CrossShardTransactionDeposit with the refund field as V2

Reply:
good call. will do

Reply:
wait, it doesn't make sense, as in line 1182 we are currently writing version 0 to all the deposits